packages/core/src/interpreter.ts

```diff
@@ -1018,6 +1080,9 @@ export class Interpreter<
     name: string,
     options?: SpawnOptions
   ): ActorRef<any> {
+    if (this.status !== InterpreterStatus.Running) {
```

Comment:
This also should not do anything if the statis is `InterpreterStatus.Stopped`

Also, when will this occur (spawning before the interpreter is started) relevant to this issue?

Reply:
Within this PR it's not as much about spawning before the interpreter is started but rather about spawning from within `exit` action where the interpreter is already stopped.